trace/tracestate_test.go

```diff
@@ -26,74 +26,91 @@ import (
 // Taken from the W3C tests:
 // https://github.com/w3c/trace-context/blob/dcd3ad9b7d6ac36f70ff3739874b73c11b0302a1/test/test_data.json
 var testcases = []struct {
+	name       string
```

Comment:
Technically, we could use `in` as the name of the subtest here. I've added an explicit name though, as I believe it makes it cleared what the case's goal is.